Makefile

```diff
@@ -29,8 +29,8 @@ STATIC := $(STATIC)
 # depending on the context.
 COCKROACH_IMAGE :=
 
-RUN  := run
-GOPATH  := $(CURDIR)/_vendor:$(CURDIR)/../../../..
+RUN := run
+export GOPATH := $(CURDIR)/_vendor:$(CURDIR)/../../../..
```

Comment:
Make is weird. Apparently it only exports variables if they were specified on the command line or imported via the environment (or explicitly specified as `export`). Do you not have `GOPATH` set in your environment?